BraveRewards/BATBraveRewards.m

```diff
@@ -100,6 +100,7 @@ - (void)setupLedgerAndAds
   NSString *ledgerStorage = [self.configuration.stateStoragePath stringByAppendingPathComponent:@"ledger"];
   self.ledger = [[self.ledgerClass alloc] initWithStateStoragePath:ledgerStorage];
   self.ledger.ads = self.ads;
+  [[self ledger] insertTestPublishers];
```

Comment:
Shouldn't be called here, probably need a better way of adding test info (i.e. calling reporting methods with a button in example app)